testssl.sh

```diff
@@ -8723,7 +8723,7 @@ certificate_info() {
           determine_trust "$jsonID" "$json_postfix" || ((ret++))
      fi
 
-     # http://events.ccc.de/congress/2010/Fahrplan/attachments/1777_is-the-SSLiverse-a-safe-place.pdf, see page 40pp
+     # https://www.eff.org/files/ccc2010.pdf, see page 40pp
```

Comment:
https://events.ccc.de/congress/2010/Fahrplan/attachments/1777_is-the-SSLiverse-a-safe-place.pdf works too

Reply:
Works for me now, did not before.